docsets/altair/docset.json

```diff
@@ -1,16 +1,21 @@
 {
     "name": "Altair",
-    "version": "4.1.0",
+    "version": "4.2.0",
     "archive": "altair.tgz",
```

Comment:
Please update the main archive as well next time. I'll update it by copying `versions/4.2.0/altair.tgz`.